docs/tutorials.md

```diff
@@ -64,7 +64,12 @@ Here is the lists of the availables tutorial for now:
 
 ## Concepts extraction
 
-**WIP**
+| Category      | **Tutorial Name**      | Notebook                                                                                                                                                           |
+|:------------- | :--------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| Labelled concept methods | CAV + TCAV           | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1iuEz46ZjgG97vTBH8p-vod3y14UETvVE) |
+|  |  |  |
+| Automatic concept extraction      | CRAFT Tensorflow          | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1jmyhb89Bdz7H4G2KfK8uEVbSC-C_aht_) |
```

Comment:
I thought this was a WIP, Then it should be indicated as such here to avoid the user confusion :)

Reply:
My understanding is that TCAV is not WIP, only "Robust TCAV" is ... (?)